calico/felix/felix.py

```diff
@@ -204,6 +207,7 @@ def main():
 
     try:
         config = Config(options.config_file)
+        _log.debug("felix cfg loaded")
```

Comment:
Perhaps it would be nice for this log to say where the config was loaded from; include options.config_file in the message?

Probably of marginal utility, but it might be useful and I see no reason not to.